### DISCUSSION TITLE
Format date according user locale with date-fns

### DISCUSSION THREAD

**Discussion by cflorez10** [js-fns/date-fns#2728] [Q&A]
I'm using `date-fns` and I would like to display dates as per user’s selected locale (language+region) for example: **“Jan 12, 2021” for en_US** or **“12 Jan 2021.” for en_CA.**, There is a way to achieve this with `date-fns`?

**Comment by fturmel** [comment]
You can pass a locale to many functions in the library, including `format`. Look for the `p` / `P` tokens available.

https://date-fns.org/v2.25.0/docs/I18n
https://date-fns.org/v2.25.0/docs/format

**Reply by KonradLinkowski** [reply]
But `format` for example requires also a `format` argument, not only the locale.
